__tests__/_extend-jest.ts

```diff
@@ -0,0 +1,41 @@
+import { FetchMock } from './_helper'
+
+export function extendExpect() {
+  expect.extend({ toHaveBeenCalledFor, toHaveBeenCalledOnceFor })
+}
+
+function toHaveBeenCalledFor(
+  this: jest.MatcherUtils,
+  mockedFetch: FetchMock,
+  url: string
+): jest.CustomMatcherResult {
+  const numberOfCalls = mockedFetch.getAllRequestsFor(url).length
+
+  return {
+    pass: numberOfCalls > 0,
+    message: () => `URL ${url} was called ${numberOfCalls} times`,
+  }
+}
+
+function toHaveBeenCalledOnceFor(
```

Comment:
grammar
```suggestion
function toHaveBeenCalledOnceWith(
```

Reply:
And possibly be explicit here (since this is for fetch anyways):
```suggestion
function toHaveExactlyOneRequestTo(
```